local_node.sh

```diff
@@ -22,7 +23,10 @@ GENESIS=$HOMEDIR/config/genesis.json
 TMP_GENESIS=$HOMEDIR/config/tmp_genesis.json
 
 # validate dependencies are installed
-command -v jq > /dev/null 2>&1 || { echo >&2 "jq not installed. More info: https://stedolan.github.io/jq/download/"; exit 1; }
+command -v jq >/dev/null 2>&1 || {
+	echo >&2 "jq not installed. More info: https://stedolan.github.io/jq/download/"
```

Comment:
Should we be using tabs instead of 2 or 4 spaces in shell scripts? I thought that was Go only

Reply:
All changes in `local_node.sh` is a result of applying `shellcheck && shfmt -l -w local_node.sh` linters commands.
Also we still have linter fail on `useless cat rule`,  but this changes out of scope of the PR